src/Components/Onboarding/Steps/Artists/PopularArtists.tsx

```diff
@@ -175,7 +176,7 @@ class PopularArtistsContent extends React.Component<Props, null> {
               key={artist.id}
               id={artist.id}
               name={artist.name}
-              image_url={artist.image && artist.image.cropped.url}
+              image_url={artist.image && get(artist.image, i => i.cropped.url)}
```

Comment:
`image_url={get(artist.image, i => i.cropped.url)}`

Reply:
It's inside `map` function so was not very obvious without changing the flow. Not even sure we need to update those. I'm not sure where this is used but figured it could never hurt to be defensive. But will update for your `artist` suggestion.